sdl_viewer/shaders/terrain.vs

```diff
@@ -0,0 +1,55 @@
+#version 410 core
+// The vertex positions are in grid coordinates, i.e. between 0 and grid_size (inclusive).
+layout(location = 0) in ivec3 aPos;
+
+// These are constant throughout the program.
+uniform double grid_size;
```

Comment:
is `grid_size` used anywhere? Also, should it have `int` type?

Reply:
No, good point!